src/themes/schulportal/login/login.ftl

```diff
@@ -14,8 +14,14 @@
 
                             <input tabindex="2" id="username" class="${properties.kcInputClass!}" name="username" placeholder="${msg("username")}" value="${(login.username!'')}" type="text" autocomplete="username" data-testid="username-input"
                                 aria-invalid="<#if messagesPerField.existsError('username','password')>true</#if>"
-                                <#if !(login.username?? && login.username != '')>autofocus
-                                <#else> readonly</#if> />
+                                <#if !(login.username?? && login.username != '')>autofocus</#if>
+                            />
+                                <script>
+                                    const params = new URLSearchParams(document.location.search);
+                                    const hint = params.get("login_hint");
+                                    if (hint)
```

Comment:
@clauyan @AlexanderUngefug @marode-cap 
```suggestion
                                    const kcAction = params.get("kc_action");
                                    if (kcAction && kcAction === 'UPDATE_PASSWORD')
```

Reply:
I'm not really a fan of doing something like this in JS, but I don't know enough about keycloaks templates to do it serverside.

That being said, I'd recommend inverting the logic (i.e. make the field editable by default and set it readonly in JS).
Just in case somebody has JS disabled or something else goes wrong

Reply:
@marode-cap Neither are we, trust me. 😄 But we need a fix quickly and could look into the FreeMarker syntax more thoroughly afterwards. If you know someone on the project with FreeMarker knowledge, please let me know.

Reply:
Since we are pressed for time on this and all simple solutions turned out to be dead-ends, we agreed with PO to remove the readonly-attribute completely and try to fix the update-flow at a later time.